React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -43,7 +43,11 @@ @implementation RCTActionSheetManager {
 
 RCT_EXPORT_MODULE()
 
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+@synthesize bridge = _bridge;
```

Comment:
i am confused. synthesize isn't required for most property declaration in objc for last couple of years.? why do we need this?

Reply:
Probably because `bridge` is marked as `@optional` in `RCTBridgeModule`. (See line 119 in RCTBridgeModule.h.)

Also, this is explicitly the recommended solution.
https://github.com/microsoft/react-native-macos/blob/e334a301c51d041680c1dcd98266ff133c18572a/React/Base/RCTBridgeModule.h#L161-L168

Reply:
that's for that reference. can you add that to your PR description as well? any place else we need to scrub through usuage of bridge and synthesize as well? what caused this regression?